internal/core/drand_proxy.go

```diff
@@ -37,8 +38,11 @@ func (d *drandProxy) Get(ctx context.Context, round uint64) (client.Result, erro
 	if err != nil {
 		return nil, err
 	}
-	// we don't need to return the metadata
+	// we don't need to return the metadata to the public
 	resp.Metadata = nil
+	// we need to set the randomness now since it isn't sent over the wire anymore in V2
+	resp.Randomness = crypto.RandomnessFromSignature(resp.GetSignature())
```

Comment:
looks like you need to make a sacrifice to the linting gods

Reply:
Wow, how comes my pre-commit hooks doing linting didn't catch that 😲

Reply:
Is your local linter the right version?